explorer/client/src/pages/object-result/ObjectResult.tsx

```diff
@@ -8,7 +8,11 @@ import ErrorResult from '../../components/error-result/ErrorResult';
 import theme from '../../styles/theme.module.css';
 import { DefaultRpcClient as rpc } from '../../utils/api/SuiRpcClient';
```

Comment:
@666lcz --> Given this code still uses the old API (`https://demo-rpc.sui.io`) how was the connection with the DevNet RPC API tested?

Reply:
Sorry I forgot to push the latest changes

Reply:
Understood. So search via inputting the correct URL now works but search using the search bar does not.

This is because the `navigateWithUnknown` function will need to be updated in `explorer/client/src/utils/api/searchUtil.ts`. The version put together here might help: https://github.com/MystenLabs/sui/blob/explore-rpc/explorer/client/src/utils/api/searchUtil.ts